integration_tests/suite/base/test_line_extension.py

```diff
@@ -98,6 +113,67 @@ def test_associate_line_and_internal_extension(line, extension):
     )
 
 
+@fixtures.line_sip()
+def test_associate_line_and_create_extension(line):
+    exten = h.extension.find_available_exten(CONTEXT)
+
+    response = confd.lines(line['id']).extensions.post(exten=exten, context=CONTEXT)
+    response.assert_created('extensions')
+    extension = response.item
+
+    try:
+        response = confd.lines(line['id']).get()
+        assert_that(
+            response.item['extensions'], contains(has_entries({'id': extension['id']}))
+        )
+    finally:
+        confd.extensions(response.item['id']).delete()
```

Comment:
The deletion works ?
Because:
https://developers.wazo.io/documentation/api/configuration.html#tag/extensions/operation/delete_extension
_An extension can not be deleted if it is associated to a line. You must delete the association first._

Reply:
Of course it does not work. I'm implementing it in the next branch :see_no_evil: